projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-following.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/**
+ * REST API endpoint for the Jetpack author recommendations block.
+ *
+ * @package automattic/jetpack
+ * @since $$next-version$$
+ */
+
+use Automattic\Jetpack\Connection\Client;
+use Automattic\Jetpack\Status\Visitor;
+
+/**
+ * Class WPCOM_REST_API_V2_Endpoint_AI
+ */
+class WPCOM_REST_API_V2_Endpoint_Following extends WP_REST_Controller {
+	/**
+	 * Namespace prefix.
+	 *
+	 * @var string
+	 */
+	public $namespace = 'wpcom/v2';
+
+	/**
+	 * Endpoint base route.
+	 *
+	 * @var string
+	 */
+	public $rest_base = 'following';
+
+	/**
+	 * WPCOM_REST_API_V2_Endpoint_AI constructor.
+	 */
+	public function __construct() {
+		$this->wpcom_is_wpcom_only_endpoint    = true;
+		$this->wpcom_is_site_specific_endpoint = false;
```

Comment:
Are you sure about this @jeherve ? I thought `wpcom_is_site_specific_endpoint` just appended the site details to the API. So instead of this being `/me/following` it would be `/sites/WPCOM_SITE/me/following`.